.ci/releaseBuild.yml

```diff
@@ -1,5 +1,5 @@
 # The name of the build that will be seen in mscodehub
-name: CompatPowerShellget-Release-$(Build.BuildId)
+name: PowerShellget-Release-$(Build.BuildId)

```

Comment:
any reason why it's not `PowerShellGet-Release-$(Build.BuildId)` i.e with the capital G? nitpick but was curious since the rest is name like so

Reply:
Likely just a typo, I can fix that